src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -20,16 +20,71 @@
 							name="auto-icon-on"
 						/>
 					</template>
+
 					<template v-slot:content>
-						<p>You'll be asked for a verification code when accessing you Kiva account.</p>
+						<div
+							v-if="isMfaActive"
+							class="two-step-verification__section"
+						>
+							<h3 class="strong">
+								2-step verification is turned on
+							</h3>
+							<p>
+								We'll ask you for your password and verification code from your
+								mobile phone in order to log into your Kiva account.
+							</p>
+							<kv-button
+								class="text-link"
+								@click.native.prevent="checkLastLoginTime"
+							>
+								Turn off 2-step verification
+							</kv-button>
+						</div>
+
+						<div
+							v-if="isMfaActive"
+							class="two-step-verification__section"
+						>
+							<h3 class="strong">
+								Your security method(s)
+							</h3>
+							<h4>Authenticator app</h4>
+							<p>Added:<span>DATE</span></p>
+							<a>REMOVE LINK</a>
+						</div>
 
-						<h3>
-							Authentication app
-						</h3>
+						<div class="two-step-verification__section">
+							<h3 class="strong">
+								{{ cardTitle }}
+							</h3>
+							<p>{{ cardSubhead }}</p>
 
-						<h3>
-							Text message or phone call
-						</h3>
+							<div class="two-step-verification__section-sub-section">
+								<h3 class="strong">
+									Authentication app
+									<span class="two-step-verification__section-sub-section-green">(Recommended)</span>
```

Comment:
I'm getting fussy, but this could just be a utility class of `class="green"`

Reply:
Not fussy, that's much cleaner. Thanks